CHANGELOG.md

```diff
@@ -1,11 +1,21 @@
 # Changelog
 
 ## Unreleased
-* Added the `Type` field to the `AgentCheck` and `HealthCheck` structures  
-* Added the possibility to specify more http request options and meta-data when registrating service checks. 
+
+## 1.6.1.2
```

Comment:
This should be `1.6.1.2-rc1`?

Reply:
We are going to make full release soon after releasing `rc1` so I think there is no need to include `rc` in the changelog.